packages/account-v2/src/hooks/useLoginHistoryData.ts

```diff
@@ -0,0 +1,82 @@
+import Bowser from 'bowser';
```

Comment:
I see this hook is used only in one place,
may I ask please to consider using https://www.npmjs.com/package/ua-parser-js instead of bowser,
I see there has been no updates for bowser for 4 years..